src/uint/lcm.rs

```diff
@@ -0,0 +1,75 @@
+//! This module implements Least common multiple (LCM) for [`Uint`].
+
+use crate::{ConcatMixed, NonZero, Uint};
+
+impl<const LIMBS: usize> Uint<LIMBS> {
+    /// Compute the least common multiple of `self` and `rhs`.
+    pub const fn lcm_uint<const WIDE_LIMBS: usize>(&self, rhs: &Self) -> Uint<WIDE_LIMBS>
```

Comment:
I'm not sure I like this `_uint` naming scheme, which I guess was introduced in #923 

It feels to me like `_uint` should be the "default" for `Self`/`Uint`, so this is just `fn lcm`

cc @andrewwhitehead